code/modules/client/preferences_savefile.dm

```diff
@@ -221,26 +223,28 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 
 
 	//Sanitize
-	asaycolor = sanitize_ooccolor(sanitize_hexcolor(asaycolor, 6, 1, initial(asaycolor)))
-	ooccolor = sanitize_ooccolor(sanitize_hexcolor(ooccolor, 6, 1, initial(ooccolor)))
-	lastchangelog = sanitize_text(lastchangelog, initial(lastchangelog))
-	UI_style = sanitize_inlist(UI_style, GLOB.available_ui_styles, GLOB.available_ui_styles[1])
-	hotkeys = sanitize_integer(hotkeys, FALSE, TRUE, initial(hotkeys))
-	chat_on_map = sanitize_integer(chat_on_map, FALSE, TRUE, initial(chat_on_map))
+	asaycolor		= sanitize_ooccolor(sanitize_hexcolor(asaycolor, 6, 1, initial(asaycolor)))
+	ooccolor		= sanitize_ooccolor(sanitize_hexcolor(ooccolor, 6, 1, initial(ooccolor)))
+	screentip_color = sanitize_ooccolor(sanitize_hexcolor(screentip_color, 6, 1, initial(screentip_color)))
+	lastchangelog	= sanitize_text(lastchangelog, initial(lastchangelog))
+	UI_style		= sanitize_inlist(UI_style, GLOB.available_ui_styles, GLOB.available_ui_styles[1])
+	hotkeys			= sanitize_integer(hotkeys, FALSE, TRUE, initial(hotkeys))
+	chat_on_map		= sanitize_integer(chat_on_map, FALSE, TRUE, initial(chat_on_map))
```

Comment:
Remove the space indenting here.

Reply:
And on the other place too below here.

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces

mfw qustinnus doesnt read contributing dot md smh

Reply:
dude asked me "whos sed" smh